gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/util/MetaFactoryUtil.java

```diff
@@ -0,0 +1,73 @@
+package uk.nhs.adaptors.pss.translator.util;
+
+import org.hl7.fhir.dstu3.model.Coding;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.Resource;
+import org.hl7.fhir.dstu3.model.UriType;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public final class MetaFactoryUtil {
```

Comment:
Just curious about the class name change from `MetaFactory` to `MetaFactoryUtil`, what was the reasoning here? 😄

Reply:
there were a few static assertions that were added, also there already existed util methods so I thought it would be good to move it under util package, hence to be consistent and follow the naming convention pattern (as well as reflect the nature of the class) I just added "util" part :)

Reply:
Thank you for the update! My main concern here is the separation of concerns. While the class primarily interacts with `Meta`, the components serve slightly different purposes.

The Factory Pattern was implemented to retrieve instances of Meta with and without security. To maintain clarity and adhere to design principles, I believe it would be beneficial to separate `MetaFactory` from `MetaUtility`. This way, the factory can focus exclusively on providing instances of Meta, while MetaUtility can handle the specific utility functions related to Meta.

Reply:
I opted in to rename it to MetaUtil as I think it would be good to keep those methods together (at least until there not too many of them).